pkg/operator/ceph/file/filesystem.go

```diff
@@ -140,9 +140,27 @@ func newFS(name, namespace string) *Filesystem {
 	}
 }
 
+// SetPoolSize method sets the sizes for MetadataPool and dataPool
+func SetPoolSize(f *Filesystem, context *clusterd.Context, clusterName, fsName string, spec cephv1.FilesystemSpec) error {
```

Comment:
Since you have `f *Filesystem`, you don't need `clusterName, fsName string`

Reply:
Make sense, because I think I can use `f.Namespace` and `f.Name`  instead of `clusterName, fsName string`